CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Default to Bolt team
+* @puppetlabs/bolt
+
+/pre-docs @MFredette @puppetlabs/bolt
```

Comment:
Is this still accurate?

Reply:
I think it will change to Chris soon, but for now I think? @MFredette ?